.gitignore

```diff
@@ -5,3 +5,7 @@
 /config.py
 /config.ini
 /config/config.ini
+/python_embeded
+/sequences
+/workflows/sequences
```

Comment:
Is this ignore needed or did you commit this by mistake?

Reply:
This comes down to my inexperiecne with PRs. I had everything set up in one location and I committed this to stop it trying to add thousands of files. When I built the PR it autmatically added the commit and I can't seem to find a way to take it out or revert it because more commits had been made. This morning I deleted the extra lines and recommitted so it should be in line with the current `main`.

I've set up a repo folder and a test folder elsewhere with all the extra stuff (python, videos, etc), so this won't be a problem going forward.